beagle/src/main/kotlin/br/com/zup/beagle/android/utils/ToolbarManager.kt

```diff
@@ -239,7 +239,7 @@ internal class ToolbarManager(private val toolbarTextManager: ToolbarTextManager
             items[i].image?.let { image ->
                 setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS)
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/ec5f3c807356edf875644ae2137bf0a8c7f0d9a5/beagle/src/main/kotlin/br/com/zup/beagle/android/utils/ToolbarManager.kt#L240"></span>Prefer "<code>SHOW_AS_ACTION_IF_ROOM</code>" instead of "<code>SHOW_AS_ACTION_ALWAYS</code>"</td>
    </tr>
  </tbody>
</table>